packages/amplify_core/lib/src/config/amplify_outputs/auth/auth_outputs.dart

```diff
@@ -0,0 +1,95 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import 'package:amplify_core/amplify_core.dart' hide MfaConfiguration, MfaType;
+import 'package:amplify_core/src/config/amplify_outputs/auth/mfa.dart';
+import 'package:amplify_core/src/config/amplify_outputs/auth/oauth_outputs.dart';
+import 'package:amplify_core/src/config/amplify_outputs/auth/password_policy.dart';
+
+part 'auth_outputs.g.dart';
+
+/// {@template amplify_core.amplify_outputs.auth_outputs}
+/// The auth outputs used by Auth category plugin
+/// to communicate with backend services.
+/// {@endtemplate}
+@zAmplifyOutputsSerializable
+class AuthOutputs
+    with AWSEquatable<AuthOutputs>, AWSSerializable, AWSDebuggable {
+  /// {@macro amplify_core.amplify_outputs.auth_outputs}
+  const AuthOutputs({
+    required this.awsRegion,
+    required this.userPoolId,
+    required this.userPoolClientId,
+    this.identityPoolId,
+    this.passwordPolicy,
+    this.oauth,
+    this.standardRequiredAttributes,
+    this.usernameAttributes,
+    this.userVerificationTypes,
+    this.unauthenticatedIdentitiesEnabled = true,
```

Comment:
Correct, the Gen 1 config does not have this field. Guest access is allowed if it's enabled on the backend. Libraries initiate the request, and upon success, they get guest credentials otherwise an error is returned. Android and iOS also ignore this value from Gen 2 config meaning no changes are needed within Amplify Auth to use this value.